TranslationMemoryManagementUtility/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Sdl.Community.TranslationMemoryManagementUtility")]
```

Comment:
AssemblyTitle & AssemblyProduct should be managed in the assemblyInfo of the individual projects